### PR TITLE
add filter functions to function list

### PR DESCRIPTION
**Pull request by StephenKappel**
in the process of adding filter functions, also updated graphing primer to better reflect new react query editor

### CODE REVIEW

**Comment by technovangelist** [comment]
That looks like a nice update. Thanks.

File: content/partials/graphingfunctions.md [hunk 1]
```diff
@@ -1,76 +1,69 @@
...
+`median_7()`           | Smoothing     | rolling median with a span of 7
+`median_9()`           | Smoothing     | rolling median with a span of 9
+`.rollup()`            | Rollup        | override default time aggregation type and time period; see the "Rollup" section below for details
+`count_nonzero()`      | Count         | count all the non-zero values
+`count_not_null()`     | Count         | count all the non-null values
+`top()`                | Rank          | select the top series responsive to a given query, according to some ranking method; see the "Top functions" section below for more details
+`top_offset()`         | Rank          | similar to `top()`, except with an additional offset parameter, which controls where in the ordered sequence of series the graphing starts. For example, an offset of 2 would start graphing at the number 3 ranked series, according to the chosen ranking metric.
+`robust_trend()`       | Filter        | fit a robust regression trend line using Huber loss; see the "Robust regression" section below for more details
+`trend_line()`         | Filter        | fit a ordinary least squares regression line through the metric values
```
Comment (githomin) [inline content/partials/graphingfunctions.md]: "an" ordinary

File: content/partials/graphingfunctions.md [hunk 2]
```diff
@@ -1,76 +1,69 @@
...
-: Retrieves lowest-valued \[5, 10, 15, 20] series using the 'last' metric.
+For example, ```bottom10_min()``` retrieves lowest-valued 10 series using the ‘min’ metric.
 
-bottom5_area, bottom10_area, bottom15_area, bottom20_area
-: Retrieves lowest-valued \[5, 10, 15, 20] series using the 'area' metric.
+**Robust regression**
 
-bottom5_l2norm, bottom10_l2norm, bottom15_l2norm, bottom20_l2norm
-: Retrieves lowest-valued \[5, 10, 15, 20] series using the 'l2norm' metric.
+The most common type of regression -- ordinary least squares (OLS) regression -- can be heavily influenced by small number of points with extreme values. Robust regression is an alternative method for fitting a regression line; it is not influenced as strongly by a small number of extreme values. As an example, see the plot below. The original metric is shown as a solid grey line. The orange dashed line is an OLS regression line, and the purple dotted line is a robust regression line. The one short-lived spike in the metric leads to the OLS regression line trending upward, but the robust regression line ignores the spike and does a better job fitting the overall flat trend in the metric.
```
Comment (githomin) [inline content/partials/graphingfunctions.md]: The most common type of ----linear---- regression

**Comment by irabinovitch** [comment]
@StephenKappel @githomin is this in production already? or should we hold on merging?

**Comment by StephenKappel** [comment]
The filter functions haven't been released yet. I'll merge this when those are GA - likely later today.

**Pull request merged**